src/vmm/tests/integration_tests.rs

```diff
@@ -123,18 +125,24 @@ fn test_dirty_bitmap_success() {
 #[test]
 fn test_disallow_snapshots_without_pausing() {
     let (vmm, _) = default_vmm(Some(NOISY_KERNEL_IMAGE));
+    let mut vm_info = VmInfo {
+        mem_size_mib: 1u64,
+        smt: false,
+        cpu_template: CpuFeaturesTemplate::None.to_string(),
+        boot_source: BootSourceConfig::default(),
+    };
 
     // Verify saving state while running is not allowed.
     // Can't do unwrap_err() because MicrovmState doesn't impl Debug.
-    match vmm.lock().unwrap().save_state() {
+    match vmm.lock().unwrap().save_state(&mut vm_info) {
         Err(err) => assert!(format!("{:?}", err).contains("NotAllowed")),
         Ok(_) => panic!("Should not be allowed."),
     };
```

Comment:
Nit: I'd suggest an improvement here like.
```suggestion
    // Verify saving state while running is not allowed.
    assert!(matches!(
        vmm.lock().unwrap().save_state(&mut vm_info),
        Err(MicrovmStateError::NotAllowed(_))
    ));
```